frontend/src/components/pages/admin/Admin.DebugBundle.tsx

```diff
@@ -82,9 +82,14 @@ export class AdminDebugBundle extends Component<{}> {
         return (
             <Box>
                 <Header />
-                {api.canDownloadDebugBundle && <Text mt={4} fontWeight="bold">Latest debug bundle:</Text>}
-                <DebugBundleLink statuses={api.debugBundleStatuses} showDeleteButton />
-                {api.debugBundleStatuses.length === 0 && <Text>No debug bundle available for download.</Text>}
+                <Box mt={4}>
+                    {(api.canDownloadDebugBundle || api.isDebugBundleExpired) && <Text fontWeight="bold">Latest debug bundle:</Text>}
+                    {api.isDebugBundleExpired && <Text>Your previous bundle has expired and cannot be downloaded.</Text>}
+                    {api.isDebugBundleError && <Text>This debug bundle was cancelled by the user and is not available for download.</Text>}
```

Comment:
This doesn't necessarily mean it was cancelled? Just because there is an error ... it _could have been cancelled_, but it _could_ have errored out for any number of other reasons.

Reply:
I agree, in Figma we have this wording but I also wanted to make it a bit more universal. Currently there is just no way to identify whether it's cancelled or it just errored for all brokers. Right now we assume that if all broker have error state, it's cancelled, but I think we need more universal wording. cc @ivpanda

Reply:
Just to clarify, if there WAS a different error, would we say what that is? E.g. something like: 

"Your debug bundle isn't available for download. [REASON]"

Reply:
@ivpanda right now we don't have a clear error message provided by the backend, at least not in case I can reproduce (when I interrupt the process). I'd go with something universal for now and we can always improve it.